build-config/conf/crosstool/gcc-4.9.2/uClibc-0.9.33.2/crosstool.armv7a.config

```diff
@@ -17,28 +17,31 @@ CT_MODULES=y
 #
 # CT_OBSOLETE is not set
 CT_EXPERIMENTAL=y
-# CT_EXPERIMENTAL_PATCHES is not set
 # CT_ALLOW_BUILD_AS_ROOT is not set
 # CT_DEBUG_CT is not set
 
 #
 # Paths
 #
 CT_LOCAL_TARBALLS_DIR="${DOWNLOADDIR}"
-# CT_SAVE_TARBALLS is not set
-CT_CUSTOM_LOCATION_ROOT_DIR=""
+CT_SAVE_TARBALLS=y
 CT_WORK_DIR="${CT_TOP_DIR}/build"
 CT_PREFIX_DIR="${XTOOLS_INSTALL_DIR}/${CT_TARGET}"
 CT_INSTALL_DIR="${CT_PREFIX_DIR}"
 CT_RM_RF_PREFIX_DIR=y
 CT_REMOVE_DOCS=y
 CT_INSTALL_DIR_RO=y
-CT_STRIP_ALL_TOOLCHAIN_EXECUTABLES=y
+CT_STRIP_HOST_TOOLCHAIN_EXECUTABLES=y
+CT_STRIP_TARGET_TOOLCHAIN_EXECUTABLES=y
 
 #
 # Downloading
 #
-CT_FORBID_DOWNLOAD=y
```

Comment:
This comment applies to all the architectures.

CT_FORBID_DOWNLOAD must be set to 'y'.

This is an old argument going back to July 2014.  See commit 749bd360896686c7edfec405b98495e184d99906.

The gist of the problem:   CT did not validate checksums on tarballs.  so we download the tarballs and verify checksums outside of CT and then tell CT to use the local tarballs.  with that approach we never want CT to download files directly.